daemon/lokinet.cpp

```diff
@@ -681,10 +681,12 @@ win32_daemon_entry(DWORD argc, LPTSTR* argv)
 
   // Report initial status to the SCM
   ReportSvcStatus(SERVICE_START_PENDING, NO_ERROR, 3000);
-  // SCM clobbers startup args, regenerate them here
-  argc = 2;
-  argv[1] = strdup("c:\\programdata\\lokinet\\lokinet.ini");
-  argv[2] = nullptr;
+  // SCM calls this function with different args than a normal "main" expects,
+  // but lokinet_main expects normal args, so set them here instead.  At the
+  // moment we are not passing any args to lokinet_main this way anyway though.
+  argc = 1;
```

Comment:
i would suggest using a totally separate array here so that we arent screwing with argc and argv